src/Phinx/Db/Adapter/AdapterWrapper.php

```diff
@@ -227,7 +227,7 @@ public function isValidColumnType(Column $column)
      */
     public function hasTransactions()
     {
-        return $this->getAdapter()->hasTransaction();
+        return $this->getAdapter()->hasTransactions();
```

Comment:
Good catch, but this should have been a separate PR. I'll make the change to master.